storage/mysql/README.md

```diff
@@ -0,0 +1,45 @@
+# MySQL Storage for Trillian Tessera
+
+This directory contains the implementation of a storage backend for Trillian Tessera using MySQL. This allows Tessera to leverage MySQL as its underlying database for storing checkpoint, entry hashes and data in tiles format.
+
+## Design
+
+See [MySQL storage design documentation](/docs/design/mysql_storage.md).
+
+### Requirements
+
+- A running MySQL server instance. This storage implementation has been tested against MySQL 8.4.
+
+## Usage
+
+### Constructing the Storage Object
+
+```go
+import (
+    "context"
+
+    tessera "github.com/transparency-dev/trillian-tessera"
+    "github.com/transparency-dev/trillian-tessera/storage/mysql"
+)
+
+func main() {
+    mysqlURI := "user:password@tcp(db:3306)/tessera"
+    db, err := sql.Open("mysql", mysqlURI)
+    if err != nil {
+        klog.Exitf("Failed to connect to DB: %v", err)
+    }
+
+    storage, err := mysql.New(ctx, db)
+    if err != nil {
+        klog.Exitf("Failed to create new MySQL storage: %v", err)
```

Comment:
Nit - you don't have an import for `klog`

Reply:
Either make sure it compiles, or make it clear that it's a snippet. This is 95% of the way to being a runnable sample.

Reply:
Good catch. I've not only added the `klog` import but also mentioned this is a snippet.